docs/development/ide-setup/intellij-setup.md

```diff
@@ -1,23 +1,27 @@
 ## *IntelliJ* Setup
 
-### Plugins:
-  - *Google Java Format*
-    - *settings > Other settings > google-java-format*
-    - Check 'enable'
+Setting up *IntelliJ* is pretty simple.  There are 4 plugins to install with the associated settings, and some settings to download and import.  
+
+### Plugins: 
+  1. *Google Java Format*
+        1. *settings > Other settings > google-java-format*
+        2.  Check 'enable'
       ![Screenshot from 2019-08-08 17-35-52
       ](https://user-images.githubusercontent.com/12397753/62746114-07cc2b80-ba03-11e9-9ac0-0b1e6e1e8788.png)
-  - *checkstyle-IDEA* [plugin](https://github.com/jshiell/checkstyle-idea)
-    - after install finish configuration in: **Other Settings > Checkstyle**
-    - select checkstyle file:
-  - *Save Actions*
-    - configure in settings to add 'final' to local variables and class variables.
-  - *Lombok*
+  2. *checkstyle-IDEA* [plugin](https://github.com/jshiell/checkstyle-idea)
+        1. after install finish configuration in: **Other Settings > Checkstyle**
+        2. select checkstyle file:
```

Comment:
Suggestion: Giving a link to the checkstyle file might make this easier?

Reply:
The 'select checkstyle file' was there before i changed anything.  I don't know what it is, so i figured i shouldn't delete it either.

Reply:
Ah, indeed it is an important step. There are two built-in checkstyle settings, you do need to explicitly select and import and the tripleA checkstyle. It's probably worth mentioning that the setting to 'scan test sources' should be selected as well so IDEA will run checkstyle on test files.

Reply:
Okay, then if I am going to do that part correctly, i will need a link to the checkstyle settings to link.  And frankly, i probably don't have them set correctly myself right now because they weren't in the document before.  

Also, is this the setting you are referring to when you say 'scan test sources'?
![checkstyle_setting](https://user-images.githubusercontent.com/54828470/96388551-339e8180-116f-11eb-9207-564c4b10b937.png)

Reply:
4 settings to change:
![Screenshot from 2020-10-18 19-18-46](https://user-images.githubusercontent.com/12397753/96394543-271e2700-1177-11eb-9460-24e2e235d60d.png)

Reply:
@CharlieAtlas I updated this documentation with that screenshot:
https://github.com/triplea-game/triplea/pull/7932

Reply:
so apparently my checkstyle isn't up to date?  or is yours an older one?  Where do i get checkstyle.xml?  i see one called checkstyle-IDEA.xml in the zip file, is that it?

Reply:
8.8 is slightly older, I think the latest is 8.30 something. Our conflict XML is a bit dated so only the older version works.

The checkstyle.xml file is checked in, you should be able to find it in the cloned code:
https://github.com/triplea-game/triplea/blob/master/config/checkstyle/checkstyle.xml

Reply:
8.36.2 is the current version.  So i need to regress to/select 8.8 then?  

I can't seem to import the settings.